docs/en/latest/plugins/http-dubbo.md

```diff
@@ -0,0 +1,127 @@
+| params_type_desc         | string  | True     |         |              | Description of the Dubbo service method signature                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| serialization_header_key | string  | False    |         |              | If `serialization_header_key` is set, the plugin will read this request header to determine if the body has already been serialized according to the Dubbo protocol. If the value of this request header is true, the plugin will not modify the body content and will directly consider it as Dubbo request parameters. If it is false, the developer is required to pass parameters in the format of Dubbo's generic invocation, and the plugin will handle serialization. Note: Due to differences in precision between Lua and Java, serialization by the plugin may lead to parameter precision issues. |
+| serialized               | boolean | False    | false   | true\|false  | Same as `serialization_header_key`. Priority is lower than `serialization_header_key`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| connect_timeout          | number  | False    | 6000    |              | Upstream tcp connect_timeout                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
```suggestion
| connect_timeout          | number  | False    | 6000    |              | Upstream tcp connect timeout                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Please update the same for `Upstream tcp read_timeout` and `Upstream tcp send_timeout ` (Chinese version included)